packager/packager.cc

```diff
@@ -1101,6 +1103,32 @@ struct PackageSpecBodyWalk {
                 info.visibleTo_.emplace_back(getPackageName(ctx, target), core::packages::VisibleToType::Normal);
             }
         }
+
+        if (send.fun == core::Names::strictDependencies()) {
+            if (send.numPosArgs() == 1) {
```

Comment:
As written, this does not ban:

- passing no args
- passing too many args
- passing keyword args
- passing a block

Reply:
The "no args" and "too many args" cases are caught by the [`enterMethod call`](https://github.com/sorbet/sorbet/pull/8207/files#diff-2c71c670c6721566158753a99c725a4826efc01a8a6a3c53a32c98fe3889252cR652), and passing a block is caught [here](https://github.com/sorbet/sorbet/blob/master/packager/packager.cc#L1210C10-L1210C21). I'll add some test cases for those.

Passing keyword args isn't banned right now though, so I'll add a check for that.

Reply:
I believe it would be the case that keyword args would be banned if you set the type of the parameter to `String` in the enterMethod call.